content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -693,7 +675,84 @@ data:
 
 - You can't use ConfigMaps for {{< glossary_tooltip text="static pods" term_id="static-pod" >}}, because the Kubelet does not support this.
 
+### Optional ConfigMaps
+
+In a Pod, or pod template, you can mark a reference to a ConfigMap as _optional_.
+If the ConfigMap is non-existent, the configuration for which it provides data in the Pod (e.g. environment variable, mounted volume) will be empty.
+If the ConfigMap exists, but the referenced key is non-existent the data is also empty.
+
+#### Optional ConfigMap in environment variables
```

Comment:
This is just an example of an optional ConfigMap, yes? The content currently makes it look like marking environment variables optional is a special case.

Maybe remove the heading and introduce this as an example:

"For example, the following Pod specification marks an environment variable from a ConfigMap as optional:"

Reply:
Sure - have updated in https://github.com/kubernetes/website/pull/35160/commits/ad33b0c10757978ffaddfa66a691dc9ea302bec1